l10n_ir_base/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Iran - Base",
+    "version": "14.0.3.0.0",
+    "author": "fadoo"
+    "Fawan",
+    "maintainer": "Saeed Raeisi",
+    "website": "https://fadoo.ir",
```

Comment:
You can add https://fadoo.ir to your DESCRIPTION.rst.
Here you need to have:
```suggestion
    "website": "https://github.com/OCA/l10-iran",
```